src/dini/conformance.js

```diff
@@ -20,7 +20,7 @@ const localizationStrings = {
         ${plural ? "are" : "is"} to be interpreted as described in
         <a href="https://www.rfc-editor.org/info/bcp14">BCP 14</a>
         ${renderInlineCitation("RFC2119")} ${renderInlineCitation("RFC8174")}
-        when, and only when, they appear in all capitals, as shown here.
+        when, and only when, it appears in all capitals, as shown here.
```

Comment:
Still need to handle plurals case (in all profiles).
```suggestion
        when, and only when, ${plural ? "they appear" : "it appears"} in all capitals, as shown here.
```

Reply:
Don't forget to run `pnpm lint --fix` after these changes!